.github/workflows/scripts/eslint_disable_check.py

```
@@ -0,0 +1,155 @@
+    parser.add_argument(
+        "--directory",
+        type=str,
+        nargs="+",
+        default=[os.getcwd()],
+        help="""One or more directories to check for eslint disable
+        statements (default: current directory).""",
+    )
+    return parser.parse_args()
+
+
+def main():
```

Ruff (0.8.2)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)